base/irrationals.jl

```diff
@@ -101,11 +101,11 @@ big(x::Irrational) = convert(BigFloat,x)
 
 ## specific irriational mathematical constants
 
-@irrational π        3.14159265358979323846  pi
-@irrational e        2.71828182845904523536  exp(big(1))
-@irrational γ        0.57721566490153286061  euler
-@irrational catalan  0.91596559417721901505  catalan
-@irrational φ        1.61803398874989484820  (1+sqrt(big(5)))/2
+@irrational π      			3.14159265358979323846  pi
+@irrational e      			2.71828182845904523536  exp(big(1))
+@irrational γ      			0.57721566490153286061  euler
+@irrational catalan_const	0.91596559417721901505  catalan
+@irrational φ      			1.61803398874989484820  (1+sqrt(big(5)))/2
```

Comment:
Ah yeah I guess I have vim set up to do that.  I can take care of this.  I should have read through Style.jl before submitting.